plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorTest.java

```diff
@@ -191,6 +191,7 @@ protected IcebergQueryRunner.Builder createQueryRunnerBuilder()
         return IcebergQueryRunner.builder()
                 .setIcebergProperties(ImmutableMap.<String, String>builder()
                         .put("iceberg.file-format", format.name())
+                        .put("iceberg.allowed-extra-properties", "extra.property.one,extra.property.two,extra.property.three,sorted_by")
```

Comment:
Which test requires `sorted_by`?

Reply:
`testIllegalExtraPropertyKey` is trying to set `sorted_by`. I wanted to check that `SUPPORTED_PROPERTIES` validation works apart from whitelisting selected properties

Reply:
I don't think readers can understand the intention without code comment.

Reply:
added comment